indico/modules/events/util.py

```diff
@@ -415,11 +415,11 @@ def get_base_ical_parameters(user, detail, path, params=None):
             'user_logged': user is not None, 'request_urls': request_urls}
 
 
-def create_event_logo_tmp_file(event):
+def create_event_logo_tmp_file(event, dir=config.TEMP_DIR):
```

Comment:
the config is not accessible at import time (i'm surprised this doesn't break!). Set it to `None` here and do `dir or config.TEMP_DIR` inside the function

Reply:
Done

Reply:
Most probably there are no calls without setting the  dir path in the call left.

Reply:
But default arguments are evaluated at import time, so it should already fail without even calling the function (and in the tests it did).